### PR TITLE
The Ember Times No. 163 - September 11th 2020

### PR DESCRIPTION
**Pull request by amyrlam**
### Addon/App releases

- [ ] [`ember-tracked-polyfill`](https://github.com/sukima/ember-tracked-polyfill) newly released ([source](https://discordapp.com/channels/480462759797063690/480499624663056390/741699604193607741))
- [ ] [`ember-cli-ui`](https://github.com/rajasegar/ember-cli-ui) newly released ([source 1](https://twitter.com/rajasegar_c/status/1297774711384846336), [source 2](https://twitter.com/rajasegar_c/status/1297775279553646597))
- [ ] [`ember-off-canvas-sidebar`](https://github.com/rajasegar/ember-off-canvas-sidebar) newly released ([source](https://twitter.com/rajasegar_c/status/1301116927247441920))
- [x] [`ember-cli-typescript`](https://github.com/typed-ember/ember-cli-typescript) v4.0 released ([list of changes](https://github.com/typed-ember/ember-cli-typescript/releases/tag/v4.0.0)) (:lock: @ijlee2) 
  > James Davis on Discord: This changes some things under the hood, but the main thing it enables is the use of the declare property modifier: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#the-usedefineforclassfields-flag-and-the-declare-property-modifier
     >
     > Note that you will need to make sure you have at least ember-cli-babel@^7.17.0 (and ember-auto-import@^1.6.0 if you are using it).
- [ ] EmberQuest, a geocaching online RPG written in Ember ([source](https://twitter.com/danmonroe/status/1296910382305599488))
  - https://emberquest.danmonroe.dev/
  - https://www.youtube.com/watch?v=Ld1xnQWkqPU
- [x] [`empress-blog`](https://github.com/empress/empress-blog) v2.0 released ([source](https://twitter.com/real_ate/status/1301536390845534208)) (:lock: @ijlee2) 
- [x] [`eslint-plugin-ember`](https://github.com/ember-cli/eslint-plugin-ember) v9.0 released ([source](https://github.com/ember-cli/eslint-plugin-ember/releases/tag/v9.0.0)) (:lock: @ijlee2) 

### Articles

- [ ] [Async Data and Autotracking in Ember Octane](https://v5.chriskrycho.com/journal/async-data-and-autotracking-in-ember-octane/)
- [x] [CI with GitHub Actions for Ember Apps (Part 2)](https://crunchingnumbers.live/2020/08/31/ci-with-github-actions-for-ember-apps-part-2/)
- [x] [Ember 3.21 Released](https://blog.emberjs.com/2020/09/02/ember-3-21-released.html) (:lock: @ijlee2) 
- [ ] [EmberJS with REST API](https://jcwebconcepts.net/emberjs-with-rest-api) (🔒 @abhilashlr https://github.com/ember-learn/ember-blog/pull/769)
- [ ] [How Does Ember's Dependency Injection System Work?](https://nullvoxpopuli.com/2020-08-08-how-does-di-work/)
- [ ] [Migrating Off of `PromiseProxyMixin` in Ember Octane](https://v5.chriskrycho.com/journal/migrating-off-of-promiseproxymixin-in-ember-octane/) (:lock: @abhilashlr)
  - Combine with the reader's question below?
- [ ] [Tracking in the Glimmer VM](https://www.youtube.com/watch?v=BjKERSRpPeI)
- [ ] [Simplify Your Ember App with Domain Components](https://mattfrankjames.com/blog/ember-domain-components/)
- [x] [Embroider preparation](https://medium.com/@kiwiupover/embroider-preparation-14d59edafc0b) ([source](https://twitter.com/kiwiupover/status/1303556480168206336)) (🔒 @jaredgalanis )
- [ ] https://ilyaradchenko.com/ember-context-and-shared-parent-state/

### Miscellaneous

- [x] Ask addon maintainers to start preparing for [Hacktoberfest](https://hacktoberfest.digitalocean.com/) (:lock: @ijlee2)
  - Read website for event details (e.g. maintainers are recommended to use the label `Hacktoberfest`)
  - TODO: Explain code of conduct (see [Learning Team's meeting notes](https://github.com/emberjs/core-notes/blob/master/learning-team/2020-08/2020-08-27.md#hacktoberfest)) for more information)
  - TODO: Find a few how-to resources for contributors or addon authors who are participating for their first time
    - [Your First Pull Request with Lyn Muldrow](https://www.youtube.com/watch?v=jZtECuvNRiw)
    - [How to Write a Good Commit Message](https://dev.to/chrissiemhrk/git-commit-message-5e21)
    - [3 Refactoring Techniques](https://crunchingnumbers.live/2020/08/08/3-refactoring-techniques/)
  - Contributors may find issues at https://help-wanted.emberjs.com/

### Readers' Questions

  - Preferred ways to replace mixins in Ember Octane (:lock: @abhilashlr may be able to cover this in a future post - Amy from speaking with locks update: pzuraq and chris krycho could help with replying / moving this forward as well), may relate: https://discuss.emberjs.com/t/migrating-off-of-promiseproxymixin-in-ember-octane/18138
  - When could we expect Embroider to be part of Ember-cli and start using it for production apps? (Amy from speaking with locks update: maybe will reach out to person directly. We do not have a RFC for using Embroider in apps, much less an estimate. Let's message direct with Times 161 writeup: https://github.com/emberjs/ember.js/issues/19099.)
  - `ember-data-model-fragments` - Does its release schedule depend on LTS release of Ember Data? (Amy from speaking with locks update: maybe will reach out to person directly and explain it's not an Ember core addon.)
  - A request for visual models (class diagram and activity diagram) for Ember Data (Amy from speaking with locks update: runspired did this for his EmberFest talk, let's connect him with the submitter)

## 📆  Future stories

- https://twitter.com/JohnRodkey/status/1300608539912794113
  > John Rodkey on Twitter: We will be blogging about the move and putting on something other than Medium so there isn't a paywall.  The main reasons, the community, batteries included, and strong conventions.
- [ ] Ask for help from the community with joining The Ember Times as a writer (let's move this one to Sep 25?)
  - TODO: Write a Twitter draft for Melanie Sumner, who can post the tweet via `emberjs`
  - TODO: Finish updating onboarding documentations

## 🤔  Questions?

See #support-ember-times on Discord or CONTRIBUTING.

## ✅  Publishing TODOs:

- [ ] Ensure no use of Markdown italic, use `<span style="font-style: italic;">` instead 
- [ ] Check Netlify preview e.g. buttons, to ensure display on emberjs.com/blog is good
- [ ] Crosspost on dev.to, delete any alex, redo any span italics, manually redo Readers Question picture, add Canonical URL 
- [ ] Set up tweet for Monday morning on tweetdeck
- [ ] Post $natural for #news-and-announcements in Discord

### CODE REVIEW

File: source/2020-09-11-the-ember-times-issue-163.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2020
+alias : "blog/2020/09/11-the-ember-times-issue-163.html"
+responsive: true
+---
+
+👋 Emberistas! 🐹
+
+Learn what's new in the Ember 3.21 release 🎉,
+Embroider preparation blog post 📖,
+get ready for Hacktoberfest 2020 👩🏽‍💻🧑🏽‍💻,
```
Comment (abhilashlr) [inline source/2020-09-11-the-ember-times-issue-163.md:13]:
```suggestion
Get ready for Hacktoberfest 2020 👩🏽‍💻🧑🏽‍💻,
```

**Review by abhilashlr — APPROVED** [review]
LGTM 🥇 🎉  I left a minor suggestion, please feel free to discard if it isn't apt.

File: source/2020-09-11-the-ember-times-issue-163.md (hunk 1 above)
Comment (amyrlam) [reply source/2020-09-11-the-ember-times-issue-163.md:13]: thanks, let's keep it as-is, is kinda one long sentence 

image

Comment (abhilashlr) [reply source/2020-09-11-the-ember-times-issue-163.md:13]: Ah! I didn't realize this. Sounds good @amyrlam

**Pull request merged**